.buildkite/pipeline.yml

```diff
@@ -1,16 +1,23 @@
 env:
   SETUP_GVM_VERSION: 'v0.5.2' # https://github.com/andrewkroh/gvm/issues/44#issuecomment-1013231151
-  ELASTIC_PACKAGE_COMPOSE_DISABLE_VERBOSE_OUTPUT: "true"
-  ELASTIC_PACKAGE_MAXIMUM_NUMBER_PARALLEL_TESTS: 3
   DOCKER_COMPOSE_VERSION: "v2.24.1"
   DOCKER_VERSION: "26.1.2"
   KIND_VERSION: 'v0.20.0'
   K8S_VERSION: 'v1.30.0'
   YQ_VERSION: 'v4.44.3'
+
+  # Agent images used in pipeline steps
   LINUX_AGENT_IMAGE: "golang:${GO_VERSION}"
+  IMAGE_UBUNTU_X86_64: "family/core-ubuntu-2204"
+  IMAGE_MACOS_ARM: "generic-13-ventura-arm"
```

Comment:
Those environment variables are already re-named in the latest changesets:
https://github.com/elastic/elastic-package/pull/2026/files#diff-a3991ebf1475eb82acab13946ffc1eca02b43917f6d5bec3898f45c8b0b9bd53R9-R14
```
  # Agent images used in pipeline steps
  LINUX_GOLANG_AGENT_IMAGE: "golang:${GO_VERSION}"
  UBUNTU_X86_64_AGENT_IMAGE: "family/core-ubuntu-2204"
  MACOS_ARM_AGENT_IMAGE: "generic-13-ventura-arm"
  # Using this one instead of family/core-windows-2022 because of conflicts with the pre-installed docker.
  WINDOWS_AGENT_IMAGE: "family/platform-ingest-beats-windows-2022"
```